### DISCUSSION TITLE
Allow invitees to suggest a date

### DISCUSSION THREAD

**Discussion by ryanbell1997** [lukevella/rallly#757] [Ideas]
I'm glad to have come across Rallly because it's so close to what I've been looking for for a while. 

The only thing missing which I think would be a decent idea is to allow and option for the host to open up the ability for guests to suggest a date.

Take this scenario, I want to meet up with 6 friends all of which I have no recent contact with and I have no idea on their calendar. Me suggesting dates is pretty pointless, but I can get it started. The guests could then suggest their best dates, and potentially sign up to any dates which have already been suggested. That way, once everyone has voted you should in theory be left with 1 or 2 best dates. 

Thoughts? Is this already implemented? Would be interested in helping implement this if there is interest.

**Comment by lukevella** [comment]
Hey @ryanbell1997, it's been suggested before. My thoughts:

What I like about it:
* It's a unique solution that has not been done before AFAIK
* Some people have asked for this specifically

What I don't like about it:
* Adds complexity for participants - Need to make it clear somehow that participants can suggest dates without distracting them from the main task.
* Adds complexity in the app - Need to differentiate between suggested dates and the primary dates. Host may need to approve suggestions. Need to limit suggestions somehow to avoid the poll from getting out of hand.

I do have a vision for Rallly that I believe caters to your use case as well. It involves introducing an availability grid (like when2meet.com) that allows a host to get a general idea of their participants' availability before proceeding to a poll. I've described this in detail in this blog post: https://rallly.co/blog/the-future-of-rallly

Let me know what you think.

**Reply by ryanbell1997** [reply]
I think your blog post makes good sense. In getting initial availability first my suggestion is essentially achieved with clearer boundary lines of the functionality.

I suppose one of the downsides to my approach is that 5 out of 6 guests might have added their availability and then the 6th guest might add a few dates that they can do, at which point those dates won't get the same viability. 

In an ideal world it would be great for everything to be sorted on one poll visit, but I'm now not sure it's possible, without a lot more complexity.

Thanks for the swift response and suggestion, I'm onboard with your thinking.
